File: java-junit5/src/test/java/com/coditory/sandbox/ClasspathSetupSpec.java
package com.coditory.sandbox;

import org.junit.jupiter.api.Test;

import static com.coditory.sandbox.base.ClasspathFileReader.readFile;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class ClasspathSetupSpec {
    @Test
    public void shouldReadATxtFileFromMain() {
        assertEquals("main-a", readFile("a.txt"));
    }

    @Test
    public void shouldReadBTxtFileFromTest() {
        assertEquals("test-b", readFile("b.txt"));
    }

    @Test
    public void shouldReadCTxtFileFromTest() {
        assertEquals("test-c", readFile("c.txt"));
    }
}

File: kotlin-junit5-no-plugin-old/src/test/kotlin/com/coditory/sandbox/ClasspathSetupSpec.kt
package com.coditory.sandbox

import com.coditory.sandbox.base.ClasspathFileReader.readFile
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ClasspathSetupSpec {
    @Test
    fun shouldReadATxtFileFromMain() {
        assertEquals("main-a", readFile("a.txt"))
    }

    @Test
    fun shouldReadBTxtFileFromTest() {
        assertEquals("test-b", readFile("b.txt"))
    }

    @Test
    fun shouldReadCTxtFileFromTest() {
        assertEquals("test-c", readFile("c.txt"))
    }
}
